src/pydocstyle/checker.py

```diff
@@ -213,8 +213,12 @@ def check_docstring_missing(self, definition, docstring):
                         else violations.D102()
                     )
                 ),
-                Function: violations.D103,
                 NestedFunction: violations.D103,
+                Function: (
```

Comment:
We shoudl also add support for method overloads and nested function overloads here?

Reply:
Hmmm, definitely support for methods is needed. Probably nested too.

Reply:
Well, this will be a bit of work to add those, I will have fun doing that over the weekend :)